docs/getting-started.md

```diff
@@ -195,9 +419,71 @@ For example, using hybrid scenario for nuc7i7dnb:
 * ACRN_EFI_GRUB2_MOD_CFG wic variable (semicolon (;) sperated list)
     to make additional entries in grub.cfg i.e insmod ext3
 
+For More information, Please check [Update Ubuntu GRUB](https://projectacrn.github.io/latest/tutorials/using_hybrid_mode_on_nuc.html?highlight=hybrid%20mode#update-ubuntu-grub)
+and [acrn-bootconf.bbclass](https://github.com/intel/meta-acrn/blob/master-next/classes/acrn-bootconf.bbclass)
+
+
+### Override distro configuration
+
+Due to parsing sequence conflict with meta-intel, weak assingments are not used in acrn-demo-sos and acrn-demo-uos distros. So to override distro configuration in local.conf, override syntex can be used i.e
+```
+WKS_FILE_acrn-demo-sos = "your-custom.wks.in"
+```
+
+
+## Build WIC Installer Image
+
+To install the image on to NUC, you could burn the .wic image to the target hardware's internal storage.
+
+Alternatively, you could build a wic based installer image where you can burn the .wic image onto USB flash drive and use USB flash drive as installer. To build the installer image for ACRN, add below lines to `conf/local.conf`:
+
+```
+BBMULTICONFIG_append  = " installer "
+```
+
+Add below in `conf/multiconfig/installer.conf`:
+
+```
+# use the installer wks file
+WKS_FILE_pn-acrn-image-base = "image-installer.wks.in"
+
+# build initramsfs to start the installation
+INITRD_IMAGE_LIVE="core-image-minimal-initramfs"
+
+# make sure initramfs and ext4 image are ready before building wic image
+do_image_wic[depends] += "${INITRD_IMAGE_LIVE}:do_image_complete"
+IMAGE_TYPEDEP_wic = "ext4"
+
+# content to be install
+IMAGE_BOOT_FILES_append = "\
+    ${KERNEL_IMAGETYPE} \
+    acrn.bin;esp/acrn.bin \
+    microcode.cpio;esp/microcode.cpio \
+    grub-efi-bootx64.efi;EFI/BOOT/bootx64.efi \
+    ${IMAGE_ROOTFS}/boot/EFI/BOOT/grub.cfg;esp/EFI/BOOT/grub.cfg \
+    ${IMGDEPLOYDIR}/${IMAGE_BASENAME}-${MACHINE}.ext4;rootfs.img \
+"
+```
+
+Now build the installer image:
+
+```
+$ bitbake mc:installer:acrn-image-base
+```
+
+> Currently ACRN WIC Installer is supported only OE-Core(poky) `master`, `hardknott` and `gatesgarth`. Support for `dunfell` is still in progress.
+
+## Tested Hardware
+The following undergo regular basic testing with their respective MACHINE types.
+
+intel-corei7-64:
+    NUC7i7DNH
+
+intel-skylake-64:
+    NUC7i7DNH
```

Comment:
So it's OK to use a SKL machine type for KBL?

Reply:
intel-skylake-64 machine should be use for Sky Lake and its successor platforms. So It is OK to use for KBL.
I have added comment in doc too.